cldf/requirements.txt

```diff
@@ -1,47 +1,58 @@
 appdirs==1.4.4
-bs4==0.0.1
-certifi==2021.10.8
-cldfbench==1.8.0
-cldfcatalog==1.3.2
-clldutils==3.10.0
-colorlog==6.5.0
-csvw==1.11.0
-gitdb==4.0.9
-greenlet==1.1.2
-idna==3.3
-iniconfig==1.1.1
-isodate==0.6.0
-lingpy==2.6.8
-Markdown==3.3.4
-networkx==2.6.3
-newick==1.3.1
-numpy==1.21.3
-openpyxl==3.0.9
-packaging==21.2
-pluggy==1.0.0
+attrs==23.2.0
+Babel==2.14.0
+bibtexparser==2.0.0b7
+bs4==0.0.2
+certifi==2024.2.2
+cldfbench==1.14.0
+cldfcatalog==1.5.1
+cldfzenodo==2.1.0
+clldutils==3.22.1
+colorama==0.4.6
+colorlog==6.8.2
+csvw==3.3.0
+exceptiongroup==1.2.0
+gitdb==4.0.11
+idna==3.6
+iniconfig==2.0.0
+isodate==0.6.1
+jsonschema==4.21.1
+-e git+https://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@github.com/lexibank/zgraggenmadang@fd9696ec78fced482d7a10393c8bee4dee9c81a5#egg=lexibank_zgraggenmadang
```

Comment:
Small thing: requirements.txt would be cleaner without the dataset being installed in the local environment (which is not necessary for building).